plugin/libclang.py

```diff
@@ -64,10 +65,31 @@ def getBuiltinHeaderPath(library_path):
 
   return None
 
-def initClangComplete(clang_complete_flags, clang_compilation_database, \
-                      library_path):
+def initClangComplete():
+
+  # import snippets
+  try:
+    # m = importlib.import_module(modulename)
+    # for v in m:
+    #   # globals[
+    exec('import snippets.' + vim.eval('g:clang_snippets_engine') + ' as current_snippets')
```

Comment:
I guess this line doesn't bring `current_snippets` into function scope in python3. Python complains about unknown `current_snippets` variable. Also there is commented out code above.